forwards_paper.tex

```diff
@@ -273,6 +273,8 @@ \subsection*{Simulation benchmarks}
 
 \jda{Maybe we should quantify the factor by which RAM use increased; it's not
 	easy to read off the figures}
+\krt{I'm not sure it really matters.  It goes from a little to a lot.  And if we do relative, then we lose a lot of
+    points again}
```

Comment:
I just meant the '2 orders of magnitude' comment I'd added in the main text; but see @petrelharp comment in email

Reply:
> Unsimplified tree sequences grow quickly, and so storing history can use arbitrarily much memory.  However, the amount of required memory is mostly determined by the interval between simplification steps, although as shown below, there is a speed--memory tradeoff: less frequent simplification reduces overall computation time.  In fact, our method would in some situations *reduce* the amount of memory required, if memory usage in the forwards simulation was dominated by the cost of maintaining neutral genetic variants -- \emph{peak} memory usage (shown in the figures) might well be the same, since adding neutral variants to an existing tree sequence takes a similar amount of memory to maintaining neutral genotypes, but this additional usage happens after the fact (as a separate step), and for a much shorter time period.